src/SIM/AdaptiveSIM.C

```diff
@@ -15,10 +15,15 @@
 #include "SIMoutput.h"
 #include "SIMenums.h"
 #include "ASMunstruct.h"
```

Comment:
Since you now include ASMu2D and ASMu3D below which both inherit ASMunstruct, there is no longer any need for this include statement.

Reply:
Still there?